Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/Java_WiimoteAdapter.java

```diff
@@ -86,13 +86,12 @@ public static int Output(int index, byte[] buf, int size)
 
 		final int LIBUSB_REQUEST_TYPE_CLASS  = (1 << 5);
 		final int LIBUSB_RECIPIENT_INTERFACE = 0x1;
-		final int LIBUSB_ENDPOINT_OUT        = 0;
```

Comment:
Even though this is a pointless bitwise calculation, do we want to leave it as an indicator?